Dockerfile.compute-node

```diff
@@ -714,6 +714,23 @@ RUN wget https://github.com/pksunkara/pgx_ulid/archive/refs/tags/v0.1.3.tar.gz -
     cargo pgrx install --release && \
     echo "trusted = true" >> /usr/local/pgsql/share/extension/ulid.control
 
+#########################################################################################
+#
+# Layer "wal2json-build"
+# Compile "wal2json" extension
+#
+#########################################################################################
+
+FROM rust-extensions-build AS wal2json-pg-build
```

Comment:
you don’t need to extend from `rust-extensions-build` because wal2json is not a rust extension.